### PR TITLE
Nirsingh power8 pal

### PR DESCRIPTION
**Pull request by nirsingh**
Changes: 
----------
configure : We are calculating the PF_MAJOR and PF_MINOR as this is not a Universal linux, PF_MAJOR will be used in naming the rpm file.

linuxrpm.py: Calculating the name for the rpm file (scx-1.6.3-73.rhel.7.ppc.rpm) so that the PF_MAJOR can be used in the FileName.

@Microsoft/ostc-devs  @jeffaco

### CODE REVIEW

File: installer/InstallBuilder/linuxrpm.py [hunk 1]
```diff
@@ -146,9 +147,14 @@ def StageAndProperlyNameRPM(self):
                 self.fullversion_dashed + '.rhel.' + \
                 str(self.variables['PFMAJOR'])  + '.' + self.variables['PFARCH'] + '.rpm'
         elif self.variables['PFDISTRO'] == 'ULINUX':
```
Comment (nirsingh) [inline installer/InstallBuilder/linuxrpm.py]: Removed the PPC check in linuxrpm.py and made the DISTRO as REDHAT in https://github.com/Microsoft/SCXcore/pull/59/commits/7088e66dcaa14b36a14ffb3c24f502d1c5f4ea23

**Comment by nirsingh** [comment]
@vimish Squashed the commits

**Comment by jeffaco** [comment]
Just a comment: @ostcdevs isn't responding because the name is wrong. It's @Microsoft/ostc-devs (just type the dash after ostc, and GitHub should resolve the mention properly). Looking at your changes now.

File: installer/InstallBuilder/linuxrpm.py [hunk 2]
```diff
@@ -146,9 +147,9 @@ def StageAndProperlyNameRPM(self):
...
-            rpmNewFileName = self.variables['SHORT_NAME'] + '-' + \
-                self.fullversion_dashed + '.universalr.' + \
-                str(self.variables['PFMAJOR'])  + '.' + self.variables['PFARCH'] + '.rpm'
+			rpmNewFileName = self.variables['SHORT_NAME'] + '-' + \
+				self.fullversion_dashed + '.universalr.' + \
+				str(self.variables['PFMAJOR'])  + '.' + self.variables['PFARCH'] + '.rpm'
```
Comment (jeffaco) [inline installer/InstallBuilder/linuxrpm.py]: So why did you change from space indenting to tab indenting here?

Is there an actual change here at all? If not, please restore indenting and eliminate this change entirely, I think it's just unnecessary churn.

**Comment by nirsingh** [comment]
@jeffaco Properly indented the unwanted changes.

File: build/configure [hunk 3]
```diff
@@ -94,6 +94,9 @@ case $uname in
 	    ppc64*)
 		PF_ARCH=ppc
 		ARCH=ppc
+		VERSION=`grep 'Red Hat Enterprise' /etc/redhat-release | sed s/.*release\ // | sed s/\ .*//`
+		PF_MAJOR=`echo $VERSION | sed s/[.][0-9]*\//`
+	        PF_MINOR=`echo $VERSION | sed s/\[0-9]*[.]//`
```
Comment (jeffaco) [inline build/configure]: These lines are using a mix of tab and space indent, please just use space indentation here.

**Comment by jeffaco** [comment]
Just to be clear, I consider the commit message to be as important as source code. If not clear, I want to know WHY something was done (often, the WAY is what I'm after when I go back to commit messages).

So a commit message like "Add one to counter x" is useless. But a commit message like "Add one to counter x to account for NULL byte" is completely useful (you're telling why).

By the same token, these repositories are customer visible. So commit messages should be clear and proper because that's often the first thing that customers see.

**Comment by nirsingh** [comment]
@jeffaco Corrected the multi line commit messages, I am new to github trying my best to come pace asap :)

File: installer/InstallBuilder/linuxrpm.py [hunk 4]
```diff
@@ -133,7 +134,8 @@ def GenerateSpecFile(self):
 
         specfile.close()
 
-
+        if self.variables['PFDISTRO'] == 'ULINUX' and self.variables['PFARCH'] == 'ppc':
```
Comment (jeffaco) [inline installer/InstallBuilder/linuxrpm.py]: I think ULINUX is normally only defined if --enable-ulinux is used to configure. Not sure on that, though.

If you build for PPC without --enable-ulinux (because you should not), is ULINIX still defined? If not, then this AND condition would never be true.

Just check this. If it is defined, I'm good. If not, please fix and I'm still good.
